flask_frozen/__init__.py

```diff
@@ -253,14 +258,20 @@ def _check_endpoints(self, seen_endpoints):
                 stacklevel=3)
 
     def _build_one(self, url):
-        """Get the given ``url`` from the app and write the matching file.
-        """
+        """Get the given ``url`` from the app and write the matching file."""
         client = self.app.test_client()
         base_url = self.app.config['FREEZER_BASE_URL']
         redirect_policy = self.app.config['FREEZER_REDIRECT_POLICY']
         follow_redirects = redirect_policy == 'follow'
         ignore_redirect = redirect_policy == 'ignore'
 
+        destination_path = self.urlpath_to_filepath(url)
+        filename = os.path.join(self.root, *destination_path.split('/'))
```

Comment:
Actually, I see this was used below. LGTM for this PR, though we should maybe look into testing on Windows.